auxx/Main.hs

```diff
@@ -69,30 +71,52 @@ runNodeWithSinglePlugin ::
 runNodeWithSinglePlugin nr (plugin, plOuts) =
     runNode nr ([plugin], plOuts)
 
-action :: HasCompileInfo => AuxxOptions -> Either (WithCommandAction AuxxMode) Text -> Production ()
-action opts@AuxxOptions {..} command = withConfigurations conf $ do
+action :: HasCompileInfo => AuxxOptions -> Either WithCommandAction Text -> Production ()
+action opts@AuxxOptions {..} command = do
     CLI.printFlags
-    logInfo $ sformat ("System start time is "%shown) $ gdStartTime genesisData
-    t <- currentTime
-    logInfo $ sformat ("Current time is "%shown) (Timestamp t)
-    (nodeParams, tempDbUsed) <-
-        correctNodeParams opts =<< CLI.getNodeParams cArgs nArgs
     let
-        toRealMode :: AuxxMode a -> RealMode EmptyMempoolExt a
-        toRealMode auxxAction = do
-            realModeContext <- ask
-            let auxxContext =
-                    AuxxContext
-                    { acRealModeContext = realModeContext
-                    , acCmdCtx = cmdCtx
-                    , acTempDbUsed = tempDbUsed }
-            lift $ runReaderT auxxAction auxxContext
-    let vssSK = unsafeFromJust $ npUserSecret nodeParams ^. usVss
-    let sscParams = CLI.gtSscParams cArgs vssSK (npBehaviorConfig nodeParams)
-    bracketNodeResources nodeParams sscParams txpGlobalSettings initNodeDBs $ \nr ->
-        runRealBasedMode toRealMode realModeToAuxx nr $
-            (if aoNodeEnabled then runNodeWithSinglePlugin nr else identity)
-            (auxxPlugin opts command)
+        runWithoutNode = do
+          rawExec Nothing opts Nothing command
+
+    let configToDict :: HasConfigurations => Production (Maybe (Dict HasConfigurations))
+        configToDict = return (Just Dict)
+
+    hasConfigurations <- case aoStartMode of
+        Automatic -> do
+            mode <- (flip catch) (\(_ :: ConfigurationException) -> return Nothing)
+                  . (flip catch) (\(_ :: ConfigurationError)     -> return Nothing)
+                  $ withConfigurations conf configToDict
+            case mode of
+                Nothing -> putStrLn ("Starting in light mode" :: Text) >> return Nothing
+                x -> putStrLn ("Starting in standalone mode" :: Text) >> return x
```

Comment:
First, I think we called this mode "with-config", not "standalone" (but I'm okay with a rename if you do it everywhere in the codebase)

Second, I'd like to see "mode" as the first word in the message, like this:

```
Mode: with-config
```

so it would be easier to find visually in the logs